proto/types/transaction_log.proto

```diff
@@ -45,8 +45,8 @@ enum TransactionLogType {
 message TransactionStructLog {
```

Comment:
Here is struct from geth
```
type StructLogRes struct {
	Pc      uint64             `json:"pc"`
	Op      string             `json:"op"`
	Gas     uint64             `json:"gas"`
	GasCost uint64             `json:"gasCost"`
	Depth   int                `json:"depth"`
	Error   string             `json:"error,omitempty"`
	Stack   *[]string          `json:"stack,omitempty"`
	Memory  *[]string          `json:"memory,omitempty"`
	Storage *map[string]string `json:"storage,omitempty"`
}
```
Could you check whether our implementation is compatible with theirs?

Reply:
Currently except `Storage` field, everything else is the same.